src/main/scala/io/iohk/ethereum/ledger/BlockRewardCalculator.scala

```diff
@@ -39,45 +42,53 @@ class BlockRewardCalculator(config: MonetaryPolicyConfig) {
 
 
   def calcBlockMinerReward(blockNumber: BigInt, ommersCount: Int): BigInt = {
-    val era = eraNumber(blockNumber)
-
-    val baseReward = calcMinerBaseReward(era)
-    val ommersReward = calcMinerRewardPerOmmer(era) * ommersCount
-    baseReward + ommersReward
+    if (blockNumber >= byzantiumBlockNumber) newRewardAfterByzantium else {
```

Comment:
Here you don't need this `if`, as miner reward is always composed from `baseReward` and `ommersReward`  even after byzantium. (only code in `else` case is needed)